test/WebJobs.Extensions.RabbitMQ.Tests/RabbitMQServiceTests.cs

```diff
@@ -13,7 +13,7 @@ public class RabbitMQServiceTests
     {
         [Theory]
         [InlineData("", "localhost", "guest", "guest", 5672, "", "localhost", "guest", "guest", 5672)]
-        [InlineData("amqp://testUserName:testPassword@11.111.111.11:5672", null, null, null, 8080, "amqp://testUserName:testPassword@11.111.111.11:5672", "11.111.111.11", "testUserName", "testPassword", 5672)]
+        [InlineData("amqp://testUserName:testPassword@11.111.111.11:5672", null, null, null, 0, "amqp://testUserName:testPassword@11.111.111.11:5672", "11.111.111.11", "testUserName", "testPassword", 5672)]

```

Comment:
Leave it at null? i think code needs to take care of picking default port

Reply:
reverted it to null. Added an additional ruleset too.